### ISSUE TITLE
OpenDCP no longer maintained?

### ISSUE THREAD

**Issue by retokromer** [amiaopensource/open-workflows#54]
I suggest to delete the `OpenDCP` entry, because sadly this software seems not longer to be maintained. Alternatively, to add a note suggesting to use instead `DCP-o-matic`.

**Comment by retokromer** [comment]
>Alternatively, to add a note suggesting to use instead `DCP-o-matic`.

Yet that would sound strange, I guess! Better to delete the entry, in my opinion.

**Comment by privatezero** [comment]
I haven't tested this with actually trying to make a DCP, but I was able to install and open it in Ubuntu 16.04 (despite most recent release being for Ubuntu 14.04) - maybe adding a note that this project appears unmaintained but _may_ (depending on local conditions) still be usable would be adequate?

**Comment by retokromer** [comment]
Thank you, @privatezero, for your feedback!

Some of my arguments are:
- On macOS the previous release 0.29 may be much more stable than the current one 0.30. That’s the reason why back in 2016 the students had to use in majority the older one for their DCP.
- The current version has been released on 17th June 2014 and is currently 97 commits behind the head.
- On the contrary, DCP-o-matic is very responsive regarding feature and improvement requests, and constantly up-to-date with new releases.
- DCP-o-matic has an ugly interface, but offers better parametrisation possibilities than the graphically more pleasant OpenDCP.
- For those reasons the 2017 students were invited to use DCP-o-matic and this year I cancelled from the slides the OpenDCP option.

Yet, of course, I will close PR and issue if I am the only one considering this step useful for the users.
